.github/workflows/build.yml

```diff
@@ -0,0 +1,17 @@
+name: GitHub Actions Demo
+run-name: ${{ github.actor }} is testing out GitHub Actions 🚀
+on: [push]
+jobs:
+  Build_rove:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+
+    steps:
+    - name: Check out the repository
+      uses: actions/checkout@v2
+
+    - name: Build docker compose
+      run: |
+        docker compose build
```

Comment:
Change it and put : docker compose run devcontainer colcon build

This will build the image and then build the project